contracts/Allocator.sol

```diff
@@ -61,24 +61,30 @@ contract Allocator is Killable, Ownable, UsingConfig, Withdrawable {
 			pendingIncrements[_metrics] == true,
 			"Not asking for an indicator"
 		);
-		address property = metrics.property();
-		uint256 share = market.issuedMetrics() /
-			MetricsGroup(config().metricsGroup()).totalIssuedMetrics();
+		Policy policy = Policy(config().policy());
+		Lockup lockup = Lockup(config().lockup());
+		uint256 totalIssuedMetrics = MetricsGroup(config().metricsGroup())
+			.totalIssuedMetrics();
```

Comment:
In here, `totalIssuedMetrics` should indicate number of assets in the entire protocol.